### PR TITLE
Misuv-4054 Update content in What you owe page

### PR DESCRIPTION
**Pull request by andriikondiuk**

### CODE REVIEW

File: app/views/WhatYouOwe.scala.html [hunk 1]
```diff
@@ -265,16 +265,32 @@
...
+
+@getPaymentsMadeBulletLinkText = @{
+    if(isAgent) messages("whatYouOwe.payments-made-bullet-agent-1.2") else messages("whatYouOwe.payments-made-bullet-1.2")
+}
+
+@getPaymentsMadeBulletText = @{
+    if(isAgent) messages("whatYouOwe.payments-made-bullet-agent-2") else messages("whatYouOwe.payments-made-bullet-2")
+}
+
+@getNoPaymentsDueText = @{
```
Comment (avelx) [inline app/views/WhatYouOwe.scala.html:280]: worth to consider to create an extension method something like: messagesIfElse(Boolean, Key1, Key2) as this pattern appears very often across project

Comment (avelx) [reply app/views/WhatYouOwe.scala.html:280]: story created => 
https://jira.tools.tax.service.gov.uk/browse/MISUV-4079

**Review by avelx — APPROVED** [review]
comments added, looks good otherwise

File: app/views/WhatYouOwe.scala.html (hunk 1 above)
Comment (andriikondiuk) [reply app/views/WhatYouOwe.scala.html:280]: sounds good, thanks

**Pull request merged**